ModifyAv.js

```diff
@@ -0,0 +1,230 @@
+                                    <Text style={styles.bottomButtonText}>Add Times</Text>
+                                )}
+
+                            </Pressable>
+                        </View>
+                        <View style={styles.bottomButton}>
+                            <Pressable
+                                style={({ pressed }) => [{
+                                    backgroundColor: pressed ? '#D8BFD8' : '#C154C1'
+                                },
+                                styles.bottomButtonText
+                                ]}>
+                                {({ pressed }) => (
+                                    <Text style={styles.bottomButtonText}>Delete Times</Text>
+                                )}
+
+                            </Pressable>
+                        </View>
+                        <View style={styles.bottomButton}>
+                            <Pressable
+                                style={({ pressed }) => [{
+                                    backgroundColor: pressed ? '#D8BFD8' : '#C154C1'
+                                },
+                                styles.bottomButtonText
+                                ]}>
+                                {({ pressed }) => (
+                                    <Text style={styles.bottomButtonText}>Set Schedule</Text>
+                                )}
+
+                            </Pressable>
+                        </View>
+                    </View>
+                </View>
+            </LinearGradient>
+        </>
+        
+    );
+}
+
+const styles = StyleSheet.create({
+    container: {
+        //backgroundColor: '#DDA0DD'
+    },
+    // header
+    header: {
+        alignItems: 'center',
+        paddingTop: 60,
+        paddingBottom: 20,
+        backgroundColor: '#880085'
+    },
+    headerText: {
+        fontSize: 25,
+        fontWeight: 'bold',
+        color: 'white'
+    },
+    // back button
+    backButton: {
+        width: 100,
+        height: 65,
+        paddingLeft: 20,
+        paddingTop: 10
+    },
+    backButtonText: {
+        fontSize: 16,
+        fontWeight: 'bold',
+        color: 'white',
+        //backgroundColor: '#C154C1',
+        borderRadius: 20,
+        textAlign: 'center',
+        paddingTop: 5,
+        paddingBottom: 5,
+        //elevation: 10,
+        shadowColor: 'black',
+        shadowOpacity: 0.1,
+        alignItems: 'center'
+    },
+    // calendar.  calendarText is placeholder
+    calendar: {
+        height: 200,
+        //backgroundColor: 'white',
+        alignItems: 'center'
+    },
+    calendarText: {
+        color: 'black'
+    },
+    // date
+    dateContainer: {
+        //backgroundColor: 'lightblue',
+        height: 35
+    },
+    dateText: {
+        color: 'white',
+        fontSize: 20,
+        fontWeight: 'bold',
+        paddingTop: 5,
+        paddingLeft: 20
+    },
+    // for the time slots
+    timeContainer: {
```

Comment:
add borderRadius to curve out the edges as seen in prototype. Over all though code looks pretty solid